cfgov/filing_instruction_guide/wagtail_hooks.py

```diff
@@ -0,0 +1,12 @@
+from wagtail.core import hooks
+
+from filing_instruction_guide.models import FIGContentPage
+
+
+def assign_content_section_ids(request, page):
+    if page.specific_class == FIGContentPage:
+        page.assign_section_ids()
+
+
+hooks.register("after_create_page", assign_content_section_ids)
```

Comment:
Per [the Wagtail documentation](https://docs.wagtail.org/en/v2.15.5/reference/hooks.html#after-create-page):

> If you set attributes on a Page object, you should also call save_revision(), since the edit and index view pick up their data from the revisions table rather than the actual saved page record.

In practice this means that when you save a new page as draft and then go to edit it, the section IDs don't show up in the edit view. They only seem to show up in the edit view once you've published once.